<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateGridDialog</class>
 <widget class="QDialog" name="CreateGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This dialog helps the user to create a new cartesian grid file adequate for estimation or simulation of variables in a samples file (point set). It does so by suggesting grid geometry from the point set bounding box and allowing the user to choose a grid resolution with an appropriate inter-cell variance given a variogram model (gammabar).&lt;/p&gt;&lt;p&gt;The created grid contains a binary variable that results in a checkerboard pattern so the user can easily assess the generated geometry in a plot window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <layout class="QVBoxLayout" name="grpGridParameters">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_5">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="btnCalcN">
              <property name="toolTip">
               <string>Compute new nx,ny,nz from the current origin and cell size values so all data is within the grid.</string>
              </property>
              <property name="text">
               <string>nx,ny,nz = </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnPreview">
              <property name="text">
               <string>Preview</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Gamma-bar analysis</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QFrame" name="frmVariogramModels">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblVarianceLoss">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0000ff;&quot;&gt;Variance Loss = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblNugget">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Nugget effect = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>117</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnRunGammaBar">
              <property name="toolTip">
               <string>Run gammabar</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="resources.qrc">
                <normaloff>:/icons/gbar16</normaloff>:/icons/gbar16</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>117</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>130</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnCreateAndClose">
        <property name="text">
         <string>Create grid and close</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDismiss">
        <property name="text">
         <string>Dismiss</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>130</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>CreateGridDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRunGammaBar</sender>
   <signal>clicked()</signal>
   <receiver>CreateGridDialog</receiver>
   <slot>runGammaBar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCreateAndClose</sender>
   <signal>clicked()</signal>
   <receiver>CreateGridDialog</receiver>
   <slot>createGridAndClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCalcN</sender>
   <signal>clicked()</signal>
   <receiver>CreateGridDialog</receiver>
   <slot>calcN()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnPreview</sender>
   <signal>clicked()</signal>
   <receiver>CreateGridDialog</receiver>
   <slot>preview()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>284</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>runGammaBar()</slot>
  <slot>createGridAndClose()</slot>
  <slot>calcN()</slot>
  <slot>preview()</slot>
 </slots>
</ui>
